charts/presto/values.yaml

```diff
@@ -239,6 +245,21 @@ worker:
   tolerations: []
   # Worker security context (overwrites default security context)
   securityContext: {}
+  
+# Enables Presto to use Velox based workers  
+velox: 
+  enabled: false
+  # Image details for Velox workers (overrides common image details)
+  image:
+    repository: prestodb/presto-native
+    pullPolicy: IfNotPresent
+    tag: ~
+  # Pull secrets for Velox workers (overrides common image details)
+  imagePullSecrets: []
```

Comment:
I don't think we need additional `imagePullSecrets` in `velox` configuration subtree.
Let's use top level `{{ .value.imagePullSecrets }}` only if there are no objections.
```suggestion
```